app/views/projects.html

```diff
@@ -97,27 +97,29 @@ <h2 class="text-center">Loading...</h2>
               <div class="list-group list-view-pf projects-list">
```

Comment:
.list-view-pf has been deprecated in PatternFly.  I wonder if we should re-evaluate its use here?  Additionally, it's adding a lot of extraneous markup that we really don't need given the simplicity of the layout.